### PR TITLE
Added one step to rollback section batch-migration-from-previous-versions.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1609.

Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1863.

Based on the following article I think this step is needed:
https://docs.microsoft.com/exchange/configure-legacy-public-folders-where-user-mailboxes-are-on-exchange-2013-servers-exchange-2013-help

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/collaboration/public-folders/batch-migration-from-previous-versions.md [hunk 1]
```diff
@@ -445,3 +445,11 @@ If you run into issues with the migration and need to reactivate your Exchange 2
...
    Set-OrganizationConfig -PublicFolderMigrationComplete $false
    ```
+   
+4. On the Exchange 2016 server, run the following command to remove the public folder mailboxes.
+
+   ```PowerShell
+   Set-OrganizationConfig -PublicFoldersEnabled Remote -RemotePublicFolderMailboxes <ProxyMailbox1>,<ProxyMailbox2>,...,<ProxyMailboxN>
+   ```
+   
+   For more information about the remote Public Folder mailboxes you must use with this command, see [Configure legacy public folders where user mailboxes are on Exchange 2013 servers](../../../ExchangeServer2013/configure-legacy-public-folders-where-user-mailboxes-are-on-exchange-2013-servers-exchange-2013-help.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/collaboration/public-folders/batch-migration-from-previous-versions.md:455]:
```suggestion
   For more information about the remote public folder mailboxes you must use with this command, see [Configure legacy public folders where user mailboxes are on Exchange 2013 servers](../../../ExchangeServer2013/configure-legacy-public-folders-where-user-mailboxes-are-on-exchange-2013-servers-exchange-2013-help.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**